tests/packagedcode/data/m2/com/jcraft/jsch/0.1.23/jsch-0.1.23.pom.package.json

```diff
@@ -30,7 +30,14 @@
   "vcs_url": null, 
   "copyright": null, 
   "license_expression": null, 
-  "declared_license": "BSD\nhttp://www.jcraft.com/jsch/LICENSE.txt\nLicense information from http://www.jcraft.com/jsch", 
+  "declared_license": [
+    {
+      "name": "BSD", 
+      "url": "http://www.jcraft.com/jsch/LICENSE.txt", 
+      "comments": "License information from http://www.jcraft.com/jsch", 
+      "distribution": "License information from http://www.jcraft.com/jsch"
```

Comment:
Same as above this is not a proper repo value.

Reply:
Yep. Fixed.